packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -241,12 +249,13 @@ export class WaferMap<
     public update(): void {
         this.validate();
         if (this.isExperimentalUpdate()) {
-            this.experimentalUpdate();
+            // eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
+            void (() => {
+                this.currentTask = this.experimentalUpdate();
+            })();
```

Comment:
Please make sure you understand the use-case of the void expression. It seems to be used for no reason here. The void expression is used for floating promises: https://typescript-eslint.io/rules/no-floating-promises/#ignorevoid

This use-case is not a floating promise. We are saving the result of the promise to be used, not completely ignoring the result.

```suggestion
            this.currentTask = this.experimentalUpdate();
```

Reply:
You are right, somehow I didn't notice I no longer needed the void expression, solved in this commit https://github.com/ni/nimble/pull/1929/commits/3c777a28161edf98ae30dc14454ba9c570c45906